app/views/layouts/_footer.html.erb

```diff
@@ -43,6 +43,8 @@
     </div>
     <div class='row'>
       <div class="col m10 offset-m1 center-align">
+        <p><a href="http://www.guidestar.org/profile/47-4247572">
+        <img src="https://camo.githubusercontent.com/ad78997b7ab7b62d4f93b59860b1e19821676b58/687474703a2f2f776964676574732e6775696465737461722e6f72672f6778696d616765323f6f3d39353232313031266c3d7633" data-canonical-src="http://widgets.guidestar.org/gximage2?o=9522101&amp;l=v3" style="max-width:100%;"></a></p>
```

Comment:
Do you have time to add this image to app/assets/images and use a rails [image tag](http://guides.rubyonrails.org/layouts_and_rendering.html#asset-tag-helpers)?

If not we can just file a ticket.